packages/core/src/Banner.js

```diff
@@ -74,7 +74,7 @@ const Banner = props => {
         )}
         <Box width={1}>
           <Text textAlign={props.textAlign}>
-            <Heading.h5 textStyle="display2">{props.header}</Heading.h5>
+            <Text textStyle="display2">{props.header}</Text>
```

Comment:
should this maybe just accept a header prop as a node, so that headers can be used if they are wanted?

@wesleymartin85 any thoughts on whether Headings might be desirable in banners?

Reply:
@sdalonzo For accessibility purposes, we don't want to include a heading inside this banner, UNLESS, there is other content inside the banner that is related to the heading.

Maybe this banner could just accept a node, to make it a bit more flexible?

<img width="784" alt="Screen Shot 2019-09-26 at 11 43 13 AM" src="https://user-images.githubusercontent.com/11232323/65703551-f098da00-e052-11e9-98f6-ec48ee0153b2.png">

Reply:
@sdalonzo Looking at the source code more closely, it looks like there is content under the header. So I think it would make sense to accept a node for the header. Maybe we can discuss this and fix this as a follow fix, as to not block this PR?

Reply:
works for me